### ISSUE TITLE
Link to changelog

### ISSUE THREAD

**Issue by wolframhaussig** [elastic/apm-agent-rum-js#561]
I was looking for the changelog of the RUM agent and came across the following URL: https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/release-notes.html

This contained the following text and link
> Release notes are published in the [changelog](https://github.com/elastic/apm-agent-rum-js/blob/master/CHANGELOG.md).

Unfortunately, this link points back to the URL I came from:
> Release notes are now available in our documentation at ([elastic.co](https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes.html))

I guess the mistake is that the Url in GitHub points to
https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes.html
but should point to 
https://www.elastic.co/guide/en/apm/agent/rum-js/master/release-notes.html

At least I found the changelog there. Can you please check that and fix the links?

**Comment by vigneshshanmugam** [comment]
@wolframhaussig Thanks for pointing out the issue. 

@bmorelli25 Can you confirm that we don't build the current release notes for every release? or should we point to master release notes?

**Comment by bmorelli25** [comment]
Hi @wolframhaussig. Thanks for bringing this up.

@vigneshshanmugam -- The reason this is happening is because we don't backport documentation changes. So `4.x` thinks release notes are still published in the `master` changelog... while master (where we changed everything) thinks release notes are now published in each version's documentation.

Tl;dr: This will all be fixed during the next release when master is force pushed to the `4.x` branch. If you'd like, I can push an interim fix to `4.x`.

**Comment by vigneshshanmugam** [comment]
@bmorelli25 We just released 4.7.0, should we need to change anything in 4.x branch or all good?

**Comment by jahtalab** [comment]
@bmorelli25 , I've just merged master into 4.x and push that branch. Please feel free to reopen if there are still issues.

**Comment by bmorelli25** [comment]
Nice! Everything looks good to me in the docs now: https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/release-notes.html.

Thanks!

**Comment by wolframhaussig** [comment]
@bmorelli25 Am I missing something? I still cannot see the release notes under https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/release-notes.html

**Comment by jahtalab** [comment]
@wolframhaussig , Using that link I can see each major versions and links to their respective changelogs. Your issue might be related to caching though, please make sure to hard reload the page.
